pkg/zson/docs/spec.md

```diff
@@ -69,19 +65,19 @@ a "descriptor" and a type according to the following form:
 ```
 #<descriptor>:<type>
 ```
-There must be a single colon between the ascii integer descriptor
-and the type definition and the integer must be composed of string of
-ascii digits 0-9 with no leading 0 for any non-zero descriptor.
-The syntax for \<type> is described by the [type grammar](#type-grammar).
+There must be a single colon between the ASCII integer descriptor
+and the type definition, and the integer must be composed of a string of
+ASCII digits `0-9` with no leading `0` for any non-zero descriptor.
+The syntax for `<type>` is described by the [type grammar](#type-grammar).
```

Comment:
There were several places in the previous version of the spec where a backslash had been used to avoid a `<` character being swallowed during rendering. This is one of the advantages of using the backticks that render into fixed width: Now we're free to use backslashes only when they're something we actually want to trickle down into the rendered HTML.